thoth-export-server/src/specification_query.rs

```diff
@@ -55,7 +55,7 @@ impl SpecificationQuery {
         // get a vector of all page offsets we will need
         let offsets = match total_pages {
             1 => vec![0], // otherwise a range of (1..1) gives us nothing
```

Comment:
Special case no longer needed as it's fixed by the below (a range of (1..1) gives nothing, but a range of (1..=1) gives [0])

Reply:
ah, of course!